src/current/v23.1/cdc-queries.md

```diff
@@ -190,6 +190,57 @@ For newly inserted rows in a table, the `cdc_prev` column will emit as `NULL`.
 If you do not need to select specific columns in a table or filter rows from a changefeed, you can instead create a changefeed using the [`diff` option]({% link {{ page.version.version }}/create-changefeed.md %}#diff-opt) to emit a `before` field with each message. This field includes the value of the row before the update was applied.
 {{site.data.alerts.end}}
 
+### Reference TTL in a CDC query
+
+In CockroachDB, table row deletes occur as a result of regular SQL transactions or through [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}). When your changefeed emits [delete event messages]({% link {{ page.version.version }}/changefeed-messages.md %}#delete-messages), you may need to distinguish between these two types of deletion. For example, only emitting messages for row-level TTL deletes from your changefeed.
+
+If you have TTL logic defined with [`ttl_expiration_expression` or `ttl_expire_after`]({% link {{ page.version.version }}/row-level-ttl.md %}#syntax-overview), you can leverage CDC queries to determine whether or not a given row was expired at the time of the changefeed event, including a delete event.
+
+When the table uses the `ttl_expire_after` storage parameter, you can emit rows that were deleted after expiring from the changefeed with syntax similar to:
+
+{% include_cached copy-clipboard.html %}
+~~~sql
+CREATE CHANGEFEED INTO 'external://sink'
+AS SELECT cdc_prev FROM test_table
+WHERE event_op() = 'delete'
+AND (cdc_prev).crdb_internal_expiration < statement_timestamp();
+~~~
+
+This changefeed statement:
+
+- Accesses the `cdc_prev` column for the previous state of the row.
+- Searches for `delete` events in that previous state.
+- Finds the TTL expiration timestamp of the deleted rows where it is earlier than the current statement timestamp.
+
+Refer to [Using `ttl_expire_after`]({% link {{ page.version.version }}/row-level-ttl.md %}#using-ttl_expire_after) for the `CREATE TABLE` statement and further details on `ttl_expire_after`.
+
+{{site.data.alerts.callout_info}}
+This will only emit rows that were deleted **after** expiring. Furthermore, consider that a transactional SQL delete during the window between the row expiring and the TTL job running will also cause this message to emit from the changefeed.
```

Comment:
transactional SQL delete could be a link to https://www.cockroachlabs.com/docs/stable/delete-data